features/cli/add_credential.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Storing AWS access credentials
+
+  In order to manage AWS access credentials
+  As an AWS IAM user
+  I want to store an AWS access credential
+
+  Scenario: Adding a long-term access credential without an MFA device
+    When I run `aws-session add credential` interactively
+    And I type "AKIAXXXXXXXXXXXX"
+    And I type "secretaccesskey1234567890"
+    And I type ""
+    And I type "default"
+    Then the exit status should be 0
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "MFA device (leave blank for none)"
+    And the stdout should contain "Credential alias"
+
+  Scenario: Adding a temporary security credential without an MFA device
+    When I run `aws-session add credential` interactively
```

Comment:
Temporary credentials from the Security Token Service (STS) can optionally include MFA.
This is how we enforced MFA on IAM Users in the Development account before we moved to a central authentication account.

As I understand it, temporary credentials come in four combinations:
1. IAM User, without MFA -- not really useful, other than hiding your long term credentials
2. IAM User, with MFA
3. IAM Role, without MFA -- think CI/CD
4. IAM Role, with MFA -- useful for cross-account human roles